components/engine-hdb/src/main/java/com/codbex/kronos/engine/hdb/processors/HDBSynonymDropProcessor.java

```diff
@@ -65,7 +67,12 @@ public void execute(Connection connection, HDBSynonymGroup synonymModel) throws
       } catch (SQLException ex) {
         String errorMessage = String.format("Drop synonym [%s] skipped due to an error: %s", synonymModel.getName(), ex.getMessage());
         CommonsUtils.logProcessorErrors(errorMessage, CommonsConstants.PROCESSOR_ERROR, synonymModel.getLocation(), CommonsConstants.HDB_SYNONYM_PARSER);
-        throw ex;
+
+        // TODO: Fix https://github.com/codbex/codbex-kronos/issues/420
+        // TODO: [HDBDD] Public synonyms are not created #420
+
+        // Temporary disabled -> To be uncommented when #420 [HDBDD] Public synonyms are not created is fixed
+        // throw ex;
```

Comment:
To be fixed with:
- #420

Reply:
Fixed with:
- https://github.com/codbex/codbex-kronos/pull/419/commits/ab1f7a0634c74fa2933822398ff14783bd77bd17